soc/arm/nxp_imx/mcimx6x_m4/Kconfig.series

```diff
@@ -14,5 +14,6 @@ config SOC_SERIES_IMX_6X_M4
 	select CPU_HAS_FPU
 	select CPU_HAS_SYSTICK
 	select CLOCK_CONTROL
+	select FLOAT
```

Comment:
I'd rather we change the default to y in a defconfig to give an application the option to override it.

Reply:
Changed to the defconfig.